app/controllers/v0/profile/personal_informations_controller.rb

```diff
@@ -21,7 +21,7 @@ class PersonalInformationsController < ApplicationController
       #   }
       #
       def show
-        response = Mvi.for_user(@current_user).profile
+        response = MVI.for_user(@current_user).profile
```

Comment:
I don't know how I feel about this. This makes MVI look like a constant or Module instead of a class. I think traditionally even class names that are an abbreviation are done as Titlecase.

Reply:
There is a `JSON` class 😄 maybe it could be MVIRecord? (alternately in the MVI docs they use 'Person' e.g. the soap methods are called FindPerson, UpdatePerson...)